core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -340,6 +366,39 @@ private[spark] class ExecutorAllocationManager(
     }
   }
 
+  /**
+   * Maximum number of executors to be removed per dra evaluation.
+   * This function executes logic only with `spark.dynamicAllocation.streaming.enabled` flag enabled
```

Comment:
this should have better description... if not streamingDRA does nothing, put some details about the algorithm or reference the class description to explain it since this seems to be the main functionality.

Reply:
will update it